### ISSUE TITLE
Bad code ref to "yhook.s" (should be "yhook.sc")

### ISSUE THREAD

**Issue by tphinney** [googlefonts/science-gothic#314]
```
INFO:ufo2ft.filters:Running DecomposeComponentsFilter on ScienceGothic-Oblique
INFO:ufo2ft:Building OpenType tables for ScienceGothic-Black
fontmake: Error: In 'ScienceGothic[YOPQ,slnt,wdth,wght].designspace': Generating fonts from Designspace failed: The following glyph names are referenced but are missing from the glyph set:
 yhook.s (first found at /Users/tphinney/Desktop/sg/new/masters/ScienceGothic-Blk.ufo/features.fea:96:51)
```

Looks like somewhere there is a reference that should be to "yhook.sc" but somehow accidentally dropped the "c" at the end, and is breaking things.

BUT, I can’t find "yhook.s" in the compiled features, or in the class definitions, or in the kerning code.

I fixed a few other small things, but I am going to call it a night now, @kateliev if you want to hunt down where this is coming from feel free, otherwise I will come back to it later in the week.

**Comment by kateliev** [comment]
Hmm, this is rather strange. Just compiled the font using fontmake and i see no evidence of that error.

**Comment by kateliev** [comment]
Now that i think of it, there was an error with the mentioned glyph. It was fixed in 4ee14aa95f4a28f7a8775d97c7c7976921eb8b83
